tkg/vc/client.go

```diff
@@ -697,6 +697,9 @@ func (c *DefaultClient) GetVirtualMachineImages(ctx context.Context, datacenterM
 		if ovaVersion, distroName, distroVersion, distroArch := c.getVMMetadata(&vms[i]); ovaVersion != "" {
 			path, _, err := c.GetPath(ctx, vms[i].Self.Value)
 			if err != nil {
+				if strings.Contains(err.Error(), "context canceled") {
```

Comment:
It 's better to check error instance `err == context.Canceled || err == context.DeadlineExceeded` or check context `ctx.Err() != nil`

Reply:
updated. thanks!